app/core/src/navigation/stacks/HotelsStack.js

```diff
@@ -10,7 +10,7 @@ import Homepage from '../../screens/hotelsStack/Homepage';
 import SingleHotelsPackageWrapper from '../../screens/hotelsStack/SingleHotelPackageWrapper';
 import NewHotelsPackageWrapper from '../../screens/hotelsStack/NewHotelsPackageWrapper';
 
-export default StackNavigator(
+const Stack = StackNavigator(
```

Comment:
We need to find a way to make this 'on-demand', because there are other views in MMB, like Insurances, where it would be needed to. But it does not seem as straightforward as it is (more) deeply nested.

Reply:
This is an entirely different problem, since in production, the tab bar is provided by native code. We would need a bridge to hide it

Reply:
Oh right, of course. I will create an issue for that.